README.md

```diff
@@ -56,19 +53,17 @@ git clone --recursive https://github.com/matrix-io/matrix-malos-wakeword.git
 cp -r matrix-malos-wakeword/assets .
 ```
 
+
 Run nodejs example and say some voice commands: `mia ring red`, `mia ring
 orange`, `mia ring clear` for example:
 
 ``` bash
 cd matrix-malos-wakeword
-git submodule init && git submodule update
 cd src/js_test
-sudo npm installt
+npm install
```

Comment:
you already tested it in Raspbian clean installation, are you secure we don't need sudo?